cmd/skopeo/inspect.go

```diff
@@ -64,10 +69,16 @@ See skopeo(1) section "IMAGE NAMES" for the expected format
 	flags.BoolVar(&opts.config, "config", false, "output configuration")
 	flags.AddFlagSet(&sharedFlags)
 	flags.AddFlagSet(&imageFlags)
+	flags.AddFlagSet(&retryFlags)
 	return cmd
 }
 
 func (opts *inspectOptions) run(args []string, stdout io.Writer) (retErr error) {
+	var (
+		rawManifest []byte
+		src         types.ImageSource
+		imgInspect  = &types.ImageInspectInfo{}
```

Comment:
Non-blocking: AFAICS this can be `imgInspect *types.ImageInspectInfo` and avoid creating the empty object.